.cirrus.yml

```diff
@@ -5,12 +5,12 @@
 # See https://confluence.atlassian.com/bitbucket/yaml-anchors-960154027.html
 
 windows_shard_template: &WINDOWS_SHARD_TEMPLATE
-  only_if: "changesInclude('.cirrus.yml', 'dev/**', 'bin/**') || $CIRRUS_PR == ''" # https://github.com/flutter/flutter/issues/41941
+  only_if: "changesInclude('.cirrus.yml', 'dev/**', 'bin/**') && $CIRRUS_PR == ''" # https://github.com/flutter/flutter/issues/41941
```

Comment:
Have Windows hostonly been migrated?  I see https://ci.chromium.org/p/flutter/builders/prod/Windows%20hostonly_devicelab_tests but it's failing.

Reply:
Reverted to use ||